jest.config.js

```diff
@@ -28,6 +28,6 @@ module.exports = {
         'assets/images/video_player/(.*).png': '<rootDir>/dist/assets/images/video_player/$1@2x.png',
     },
     transformIgnorePatterns: [
-        'node_modules/(?!(react-native|@react-native|jail-monkey|@sentry/react-native|react-navigation|@react-native-community/cameraroll))',
+        'node_modules/(?!(react-native|@react-native|jail-monkey|@sentry/react-native|react-navigation|@react-native-community/cameraroll|hast-util-from-selector|hastscript|property-information|hast-util-parse-selector|space-separated-tokens|comma-separated-tokens|zwitch))',
```

Comment:
can you explain why this changes are required ?

Reply:
This happens when a module is not transformed by the typescript compiler. Jest gives following error:

```
    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/en/ecmascript-modules for how to enable it.
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:
    export function zwitch(key, options) {
    ^^^^^^

    SyntaxError: Unexpected token 'export'

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1350:14)
      at Object.<anonymous> (node_modules/hast-util-from-selector/index.js:22:1)
```

So the modules need to be forced to be transformed by jest.

You can read more on this in following thread: https://stackoverflow.com/questions/49263429/jest-gives-an-error-syntaxerror-unexpected-token-export